docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -64,31 +74,114 @@ Over the duration of a given microsimulation run, simulant attributes are dynami
+
+Time-varying rates dependent on multiple factors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+e.g., Multiple Myeloma model
+
+Incorporating uncertainty
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Our microsimulation framework is formulated to routinely propagate forward the uncertainty captured by these draws. Monte Carlo methods allow us to effectively incorporate different kinds of uncertainty into our models. 
+
+Monte Carlo microsimulation methods provide a useful approach to modeling stochastic uncertainty in a conceptually valid manner because chance – that is, reliance on randomly generated numbers – is used to govern movement between health states for individual simulants from a single computer-generated population, as opposed to comparing the role of chance across multiple populations with differing characteristics.
+
+In the context of healthcare estimates typically characterized by considerable variance across quantities of interest, the use of Monte Carlo methods to run simulations using iterative draws from probability distributions provides an effective means of incorporating parameter uncertainty. Simulation methods have the added advantage of reducing confounding variance between different scenarios by ensuring that the simulated population is identical across all scenarios analyzed. The only difference is the presence or absence of health interventions. As a further refinement of these methods, microsimulations represent each person in the population as an individual simulant with attributes variably assigned across simulants to reflect the population of interest. By incorporating heterogeneity at the level of the individual simulant, microsimulations are able to provide an additional degree of flexibility and detail (such as stratifying results according to demographic or epidemiological attributes) not available in population-level models. 
+
+- Surfaces complications that are more hidden in non-individual-based models (e.g., compartmental models can obscure certain limitations)
+
+Disadvantages of microsimulation
+++++++++++++++++++++++++++++++++
+
+Resource requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+Our microsimulation models require a significant amount of detailed data to accurately represent individual behaviors and the relationships between them. Obtaining this data can be challenging and time-consuming, and there may be privacy concerns associated with using such detailed personal data. 
+
+If we're not modeling the sort of complexity that microsimulation enables... perhaps a multiplication model would suit us just as well?
+
+Residual confounding
+~~~~~~~~~~~~~~~~~~~~
+
+Getting realistic, individual-level characteristics from population-level data is difficult, and creates opportunity for residual confounding...
+
+Simulant agency
+~~~~~~~~~~~~~~~
+
+Unlike agent-based models, microsimulations do not necessarily program agency into individual simulants. This removes some of the real-world likeness of these models...
```

Comment:
As mentioned above, this feels like the only one in this list that contrasts microsimulation with ABM. Also, full disclosure, I don't really understand it, though I know I've heard Abie say it a couple times -- do you have a reference about this to help me/others grasp it?